jina/helloworld/multimodal/flow-index.yml

```diff
@@ -1,64 +1,39 @@
 jtype: Flow
 version: '1'
+py_modules:
+  - ${{ ENV.PY_MODULE }}
 executors:
   - name: segment
-    uses:
-      jtype: Segmenter
-      metas:
-        workspace: ${{ ENV.HW_WORKDIR }}
-        py_modules:
-          - ${{ ENV.PY_MODULE }}
+    uses: Segmenter
+    uses_metas:
+      workspace: ${{ ENV.HW_WORKDIR }}
   - name: craftText
-    uses:
-      jtype: TextCrafter
-      metas:
-        py_modules:
-          - ${{ ENV.PY_MODULE }}
+    uses: TextCrafter
   - name: encodeText
-    uses:
-      jtype: TextEncoder
-      metas:
-        py_modules:
-          - ${{ ENV.PY_MODULE }}
+    uses: TextEncoder
   - name: textIndexer
-    uses:
-      jtype: DocVectorIndexer
-      with:
-        index_file_name: "text.json"
-      metas:
-        workspace: ${{ ENV.HW_WORKDIR }}
-        py_modules:
-          - ${{ ENV.PY_MODULE }}
+    uses: DocVectorIndexer
```

Comment:
I prefer to have the style of `using` `with` and `metas` than the `uses_with` and `uses_metas`. 

I prefer to have each Executor as its own YAML config to be honest. `uses_with` and `uses_metas` is kind of a way to override but the other one should be preferred